### ISSUE TITLE
LineIn 

### ISSUE THREAD

**Issue by RobiDobo** [WerkAG/ESP32Radio_PCB_v3.1#8]
How do you connect the vs1053 on the pcb? I have the LINE IN pins sticking out.

**Comment by WerkAG** [comment]
The easy way is using a small cable with a male jack on both ends. LINE OUT of the VS1053 to LINE IN on the board. See the pictures, they illustrate it better than my words.

**Comment by RobiDobo** [comment]
Thank you but I still can't see what to connect it to. LINE IN of vs1053 should go to Line out on the board right? But I can't see anything labelled as "line out" or "line in" on the pcb. Pls help I'm confused where to look even after reading several times :)

**Comment by WerkAG** [comment]
The 3.5mm stereo jack on the board is the LINE IN. The 3.5mm jack on the VS1053 in LINE OUT. Just connect the two with a small 3.5mm stereo male-male cable 

Note: The reason for having the small the 3.5mm stereo jack on the board is to allow the device to work with other sound sources than just the VS1053. It will amplify or transmit by Bluetooth whatever you put on the LINE IN of the board.

Sorry for my poor English.

**Comment by RobiDobo** [comment]
Ah okay I see, I will do that but there are also 2 pins on the VS1053 beneath which is written LINE IN and I don't see anything in the scheme what to do with those. Don't worry, your english is very clear.

**Comment by WerkAG** [comment]
The line in pins on the VS1053 (decoder board) aren't used for anything.

**Comment by RobiDobo** [comment]
then should I cut them or how do i fit the vs1053 on the pcb? use female headers and leave those pins sticking out?

**Comment by RobiDobo** [comment]
Thank you very much for bearing with me :)

**Comment by WerkAG** [comment]
Yes, use female headers on the PCB, then fit the VS1053 module on them. The VS1053 module will be removable.
I suggest to also use female headers for the microcontroller.
:-)

**Comment by RobiDobo** [comment]
Thank you!
